Timer/clock.go

```diff
@@ -0,0 +1,74 @@
+package clock
+
+import (
+	"consumer"
+	"fmt"
+	"time"
+
+	database "github.com/pikapika/database"
+	service "github.com/pikapika/notification_services"
+)
+
+func eventToConsumerEvent(event database.Event) consumer.Event {
+	return consumer.Event{
+		Message:            event.Message,
+		Source:             event.Source,
+		DestinationAddress: event.DestinationAddress,
+		Topics:             event.Topics,
+		NotificationType:   event.NotificationType,
+		Status:             event.Status,
+		NextRetry:          event.NextRetry,
+		Attempts:           event.Attempts,
+	}
+}
+
+func RetryClock() {
+	for {
+		events, err := database.GetIncompleteEvents()
+		if err != nil {
+			fmt.Print("Error getting incomplete events", err)
```

Comment:
use log.Error, log.Info instead of fmt.Printl - P2

and this holds true across all the logs